File: Programming 2/Labs/Store in Python/CashRegister.py
class CashRegister:
    def __init__(self):
        self.cash = 0

    def add(self, n):
        self.cash += n
    
    def formatted(self, value):
        integer = int(value)
        Int_part = int((value - integer) * 100)
        return '${:d}.{:02d}'.format(integer, Int_part)

    def to_string(self):
        if self.cash < 1:
            return "Cash register: empty"
        else:
            return 'Cash register: {}'.format(self.formatted(self.cash))

File: Programming 2/Labs/Store/CashRegister.java
import java.text.*;


public class CashRegister {
    private double cash;

    public CashRegister() {
        this.cash = 0;
    }

    public void add(double n) {
        this.cash += n;
    }

     public String formatted (double value) {
        DecimalFormat f = new DecimalFormat("###,##0.00");
        return f.format(value);
     }
    @Override
    public String toString() {
        if (cash < 1) { 
            return "Cash register: empty";
        }
        else {
            return "Cash register: $" + formatted(this.cash);
        }
        
    }
}
